src/wasm/literal.cpp

```diff
@@ -989,7 +1023,7 @@ Literal Literal::div(const Literal& other) const {
         case FP_ZERO:
           switch (std::fpclassify(lhs)) {
             case FP_NAN:
-              return Literal(setQuietNaN(lhs));
+              return standardizeNaN(lhs / rhs);
             case FP_ZERO:
               return Literal(
                 std::copysign(std::numeric_limits<float>::quiet_NaN(), sign));
```

Comment:
Should this NaN be standardized as well?

Reply:
Done.